src/components/expansion-panel/react/ExpansionPanel.tsx

```diff
@@ -115,15 +116,14 @@ const ExpansionPanel: React.FC<ExpansionPanelProps> = (props: ExpansionPanelProp
         setShowWrapper(shouldOpen);
     };
 
-    const color = theme === Theme.light ? ColorPalette.dark : ColorPalette.light;
+    const color = theme === Theme.dark ? ColorPalette.light : ColorPalette.dark;
```

Comment:
😂

Reply:
Yeah if the theme is dark, the font is light and otherwise the font is dark